test/specs/dom/index.js

```diff
@@ -361,9 +361,9 @@ if (typeof window !== 'undefined') {
                 describe('palindrom-morph-url event', function() {
                     beforeEach(async () => {
                         // wait for Palindrom to call .listen (after finishing the ajax request)
-                        await sleep();
+                        await sleep(10);
```

Comment:
This change actually gives **less** time for waiting, because the default sleep value is `30`. See: https://github.com/Palindrom/Palindrom/blob/a5869cd879fd7a37848f8733f5bc911db391a12e/test/utils/index.js#L1

Reply:
ha! thanks, I'll give it `100` and hope that will solve the flakiness 😕